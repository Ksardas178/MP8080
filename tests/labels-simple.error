line 11: <ERROR> first argument must be B, D, C or SP register specified
