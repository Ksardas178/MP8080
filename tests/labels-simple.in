MOV 1 2
MOV B C
JZ 23 45
JZ 132 29
>LABEL_1
JZ d23 45
JZ d21 126+21-12*2
JZ o23 1++
>LABEL_2
JZ b1100011-- ++2
LXI 3 LABEL_1
>LABEL_3
LXI B LABEL_2
CALL LABEL_3
