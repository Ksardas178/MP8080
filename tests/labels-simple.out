
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: MOV 1 2
00000001: MOV 0 1
00000010: JZ 23 45
00000011: JZ 132 29
00000100: JZ 23 45
00000101: JZ 21 123
00000110: JZ 19 2
00000111: JZ 99 2
00001000: LXI 3 4 0
00001001: LXI 0 7 0
00001010: CALL 9 0
